src/stories/components/DataCard/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Data Card
+
+.data-card {
+    border-radius: 5px;
+}
+
+.data-card__container {
+    padding: $spacing-3;
+}
+
+.data-card__content {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.data-card__title {
+    font-weight: 700;
+}
+
+.data-card__details {
+    margin-bottom: $spacing-5;
+    p {
+        margin-bottom: $spacing-1;
+        font-size: $font-size-md;
+        font-family: $header-font-family;
+        line-height: $line-height-tight;
```

Comment:
You can apply this directly to the `.card` class and it will automatically apply it to the paragraph.